intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -125,7 +125,7 @@ Argument description of WeightOnlyQuantConfig:
 | weight_dtype      | String      | Data type of quantized weight: int4/int8 (default int4)                                 |
 | alg               | String      | Quantization algorithm: sym/asym (default sym)                                          |
 | group_size        | Int         | Group size: Int (default: 32)                                                           |
-| scale_dtype       | String      | Data type of scales: fp32/bf16 (default fp32)                                           |
+| double_quant_scale_dtype       | String      | Data type of scales: fp32/bf16 (default fp32)                                           |
```

Comment:
any more specific explain like what is double-quant?
A link to reference will be appreciate.